### DISCUSSION TITLE
z-index on walletconnect

### DISCUSSION THREAD

**Discussion by quantix-nft** [wevm/wagmi#2775] [Q&A]
Hello, I'm using wagmi to handle my defi connection

`const config = createConfig({
  autoConnect: true,
  connectors: [
    new MetaMaskConnector({ chains,options: {
                                       shimDisconnect: true,
                                     }, }),

    new WalletConnectConnector({
      chains,
      options: {
        projectId: process.env.REACT_APP_WALLETCONNECTPROJECTID,
      },
    }),

  ],
  publicClient,
  webSocketPublicClient,
})`

The problem occurs when I use walletconnect connector: the form that pops up (wallet connect with qr-code and wallets to use) has a too low z-index and I don't know how to fix it.

Please help !

**Comment by SaadNiaziDev** [comment]
Are you using web3modal? Or just wallectConnectConnector by wagmi?

**Reply by quantix-nft** [reply]
hello, only wallectConnectConnector

**Reply by SaadNiaziDev** [reply]
maybe you need to overwrite their css
`--w3m-z-index` 
this is the class that is needed to override

**Comment by glitch-txs** [comment]
in options you can add this:
```ts
    qrModalOptions:{
      themeVariables:{
        "--wcm-z-index": "9999999"
      }
    }
```

Full list [here](https://docs.walletconnect.com/2.0/web/walletConnectModal/modal/theming#list-of-theme-variables) :)

**Reply by liringlas** [reply]
Thank you very much!
